test/Sentinel1Test.jl

```diff
@@ -4,7 +4,7 @@ import .Sentinel1
 
 @testset "Sentinel1.jl" begin
      
-    ####### Generate test data ##############
+    ####### Helper function ##############
     createMetaData() = Sentinel1.MetaDataSLC("VV",1,Sentinel1.DateTime(2013,7,1,12,30,59),5405)
     
     function createSwathSLC() 
```

Comment:
Is there missing an "end" here?

Reply:
@eyhl 
I am not sure what "end" you mean. Is it for the `@testset "Sentinel1.jl" begin`? 
I have just made a commit where I move that line down so it is easier to read. 

The unit tests would fail if I was missing an `end` and they are passing

Reply:
for 
`function createSwathSLC() `
(maybe I am just a noob, and oversaw something)

Reply:
I think I just got confused about the layout in GitHub. 😆 Sorry, everything is fine.